makefiles/tools/serial.inc.mk

```diff
@@ -19,4 +19,7 @@ ifeq ($(RIOT_TERMINAL),pyterm)
 else ifeq ($(RIOT_TERMINAL),picocom)
     export TERMPROG  ?= picocom
     export TERMFLAGS ?= --nolock --imap lfcrlf --echo --baud "$(BAUD)" "$(PORT)"
+else ifeq ($(RIOT_TERMINAL),miniterm.py)
+    export TERMPROG  ?= miniterm.py
+    export TERMFLAGS ?= --eol CRLF "$(PORT)" "$(BAUD)"
```

Comment:
Maybe also enable `--echo` ? It's the kind of useful things when using the shell.

Reply:
With local echo you end up having _double_ echo, as the shell echoes too, and miniterm does not do line buffering, eg.:

```
> hheellpp
```

Reply:
Having local echo messes with the tests also as the sent string is matched by pexpect.